src/components/instance/browse/BrowseDatatableHeader.js

```diff
@@ -64,7 +64,7 @@ function BrowseDatatableHeader({
 					onClick={toggleOnlyCached}
 				>
 					<span className="me-2">only cached</span>
-					<i className={`fa fa-lg fa-toggle-${onlyCached ? 'on' : 'off'}`} />
+					<i className={`fa fa-lg fa-toggle-${!onlyCached ? 'on' : 'off'}`} />
```

Comment:
Does this change the default or just change how the button displays the value (seems like it reversing how it shows the value and shows in incorrectly reversed)?
I thought you needed to change the default here: https://github.com/HarperDB/hdbms/blob/master/src/components/instance/query/Datatable.js#L28